virtcontainers/network.go

```diff
@@ -448,13 +454,11 @@ func xconnectVMNetwork(endpoint Endpoint, connect bool, numCPUs uint32, disableV
 	}
 	switch netPair.NetInterworkingModel {
 	case NetXConnectBridgedModel:
-		netPair.NetInterworkingModel = NetXConnectBridgedModel
```

Comment:
Isn't it breaking a current behavior? Or did you remove it because it was not needed anymore?

Reply:
See https://github.com/kata-containers/runtime/pull/733#discussion_r223558592. IMO it is a redundance

Reply:
Looks fair to me (to remove it I mean), just please make sure there's not some weird behavior related to it that we're missing.